src/bin/vip-import-media.js

```diff
@@ -66,15 +92,68 @@ command( {
 	appContext: true,
 	appQuery,
 	envContext: true,
-	requiredArgs: 1,
 	module: 'import-media',
-	requireConfirm: 'Are you sure you want to import the contents of the provided file?',
-	skipConfirmPrompt: true,
+	requireConfirm: `
+${ chalk.red.bold( 'NOTE: If the provided archive\'s directory structure begins with `/wp-content/uploads`,' ) }
+${ chalk.red.bold( 'we will extract only the files after that path and import it. Otherwise, we will' ) }
+${ chalk.red.bold( 'import all files and preserve the directory structure as is.' ) }
+
+Are you sure you want to import the contents of the url?
+`,
 } )
-	.command( 'status', 'Check the status of the current running import' )
+	.option( 'url', 'Valid URL to download a file archive from', '' )
```

Comment:
@hanifn , actually I think this was incorrect. We have essentially removed the `import media status` command here

Reply:
Sorry for not detecting it earlier!

Reply:
Fixing here: https://github.com/Automattic/vip/pull/715